src/ttkbootstrap/tooltip.py

```diff
@@ -40,6 +40,7 @@ def __init__(
         text="widget info",
         bootstyle=None,
         wraplength=None,
+        waitime=500,    # milliseconds
```

Comment:
I like this. To be consistent with similar APIs, please change `waittime` to `delay`.

Reply:
sure thing! check my last commit :-)